truss/tests/test_truss_handle.py

```diff
@@ -24,6 +26,38 @@ def test_server_predict(custom_model_truss_dir_with_pre_and_post):
     assert resp == {'predictions': [4, 5, 6, 7]}
 
 
+def test_readme_generation_int_example(custom_model_truss_dir_with_pre_and_post):
+    sc = TrussHandle(custom_model_truss_dir_with_pre_and_post)
+    readme_contents = sc.generate_readme()
+    readme_contents = readme_contents.replace('\n', '')
+    readme_correct_path = Path(__file__).parent.parent / 'test_data' / 'readme_int_example.md'
+    with open(readme_correct_path, 'r') as readme_correct_file:
+        readme_correct = readme_correct_file.read().replace('\n', '')
+    assert readme_contents == readme_correct
+
+
+def test_readme_generation_no_example(custom_model_truss_dir_with_pre_and_post_no_example):
+    sc = TrussHandle(custom_model_truss_dir_with_pre_and_post_no_example)
+    # Remove the examples file
+    os.remove(sc._spec.examples_path)
+    readme_contents = sc.generate_readme()
+    readme_contents = readme_contents.replace('\n', '')
```

Comment:
Do we need to replace '\n' with '' for comparison? What if we kept it there here as well as from where we read it from the test readme file?

Reply:
Yeah the issue is that without the replacements on both READMEs, the assertion fails. When generating the README string, it contains explicit '\n' in the string but when reading from the test README file, the newline characters are interpreted as line breaks. 

<img width="1283" alt="Screen Shot 2022-07-08 at 10 56 41 AM" src="https://user-images.githubusercontent.com/48742992/178045385-ba7d2ea1-660d-42d8-9cb3-9594c055256b.png">